pkg/sac/scope_checker.go

```diff
@@ -107,12 +99,6 @@ func (c scopeChecker) TryAnyAllowed(subScopeKeyss [][]ScopeKey) TryAllowedResult
 // AnyAllowed checks if access to any of the given subscopes is allowed.
 func (c scopeChecker) AnyAllowed(ctx context.Context, subScopeKeyss [][]ScopeKey) (bool, error) {
```

Comment:
Between this and `TryAnyAllowed`, you really only need one (can be done in a separate PR)

Reply:
Also note that you can get rid of a bunch of context parameters

Reply:
Fixed in https://github.com/stackrox/stackrox/pull/2777